src/EditorFeatures/Core.Wpf/Completion/Presentation/CompletionPresenterSession.cs

```diff
@@ -12,6 +12,7 @@
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
 using Roslyn.Utilities;
+using CACompletionItem = Microsoft.CodeAnalysis.Completion.CompletionItem;
```

Comment:
:bulb: Move this inside the namespace declaration and then you can make it this:

```
using CompletionItem = Microsoft.CodeAnalysis.Completion.CompletionItem;
```

The lack of a rename makes this a less intrusive change.

(Applies to other files as well)

Reply:
fixed